pkg/config/config_template.yaml

```diff
@@ -1011,6 +1011,12 @@ api_key:
 #
 # dogstatsd_socket: ""
 
+## @param dogstatsd_telemetry_endpoint - string - optional - default: ""
+## Listen to an extra socket to receive telemetry from clients. Use
```

Comment:
This only allows the telemetry endpoint to be enabled for one transport at a time. This might be OK for now, but worth documenting the limitation here.

Also, let's make this immediately understandable and mention:
* that clients send telemetry metrics (if they send telemetry metrics) by default to the main dogstatsd destination, in which case this option can be left disabled.
* that clients need to support sending telemetry metrics to a different destination and be configured accordingly for this option to be useful.
* in which cases enabling telemetry on a different listener can be useful (you mentioned this in the release note, let's mention it here as well since this is the documentation that users will read first)

If this explanation gets a little too long for the config template, we can move it to a docs page and link it from here.

Reply:
WDYT of the new version ?